test/src/unittests/stats/test_minmax.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+from essentia.standard import MinMax
+
+from essentia_test import TestCase, TextTestRunner, allTests
+
+class TestMinMax(TestCase):
+
+    def testDefaultIsMin(self):
+        """Making the Algorithm with no type defaults to min"""
+        value, index = MinMax()([3,1,2])
+        self.assertEqual(value, 1.0)
```

Comment:
value is always floating point, even if the inputs are only integers. Is this OK?

Reply:
The input to the algorithm is `std::vector<Real>` so those values get converted and we can't expect much more.